src/zoid/buttons/util.js

```diff
@@ -396,7 +396,7 @@ export const getModal: (
 
     return window[namespace].MessagesModal({
       account: `client-id:${clientID}`,
-      merchantId: merchantID?.join(",") || undefined,
+      merchantId: merchantID?.join() || undefined,
```

Comment:
did you mean to drop the `","` in the join?

Reply:
i think by default it makes it comma delimited but i can add it back in for clarity

Reply:
wow i had no idea: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
i went ahead and added it back in just in case

Reply:
i addressed the comments